org.kde.koko.json

```diff
@@ -0,0 +1,89 @@
+{
+    "id": "org.kde.koko",
+    "rename-icon": "koko",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "koko",
+    "tags": ["nightly"],
+    "desktop-file-name-suffix": " (Nightly)",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=home",
+        "--env=XDG_DATA_DIRS=/usr/share:/app/share/"
+    ],
+    "separate-locales": false,
+
+    "modules": [
+        {
+            "name": "exiv2",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.exiv2.org/builds/exiv2-0.27.2-Source.tar.gz",
```

Comment:
you should use 0.27.3

(it needs a couple of patches to build for aarch64, like in https://github.com/flathub/org.darktable.Darktable/)

Reply:
Thanks, I made the modifications.